streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -789,6 +790,15 @@ private static ScheduledExecutorService maybeCreateRocksDBMetricsRecordingServic
         return null;
     }
 
+    private static void maybeWarnAboutCodeInRocksDBConfigSetter(final Logger log,
+                                                                final StreamsConfig config) {
+        if (config.getClass(StreamsConfig.ROCKSDB_CONFIG_SETTER_CLASS_CONFIG) != null) {
+            log.warn("RocksDB's version will be bumped to version 6+ via KAFKA-8897 in a future release. "
```

Comment:
Personally I think we should call out the specific method that was removed so we don't cause undue panic. It was only a single method, after all.